Add Pandas sum codebyte (#1971)
Codecademy/docs: content/pandas/concepts/groupby/terms/sum/sum.md

The **`.sum()`** method produces a new `Series` or [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) with aggregate sums for the groups in a [`GroupBy`](https://www.codecademy.com/resources/docs/pandas/groupby) object.

> ## Syntax 

Reviewer: ```suggestion
## Syntax
```

> ## Syntax for Pandas .sum() Method

Reviewer 2: ```suggestion
## Syntax 
```

Reviewer 2: We generally don't specify the method in the header

1   B   13.0
2   C   12.0
```

> ### Try It Yourself

Reviewer 2: ```suggestion
## Codebyte Example
```

Reviewer 2: I believe "try it yourself" is the tag line used on another site. We generally stick to "Codebyte Example"

> Use the Codebytes editor below to produce a new Series using Pandas .sum() method.

Reviewer 2: ```suggestion
Use the Codebytes editor below to produce a new Series with the `.sum()` method.
```
